exercise_2/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.14)
+project(exercise_2)
+
+set(CMAKE_CXX_STANDARD 17)
+
+add_executable(exercise_2 main.cpp IRecord.hpp IPerson.hpp Teacher.cpp Teacher.hpp RecordsHandler.cpp RecordsHandler.hpp Student.cpp Student.hpp Course.cpp Course.hpp Exam.cpp Exam.hpp)
```

Comment:
Yes, I know that. But it helps CLion IDE to manage my project. Is that bad style?

Reply:
In that case it's not a bad style.
However I'd raise a comment about adding headers into executable during every review. Looks like it might be an issue related to IDE configuration (sadly, haven't been working much with CLion - can't help with that).